packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -175,9 +192,18 @@ function AuthOptions({
     onSuccessfulLogin: onLoginCheck,
     ...(!isTesting && { queryEnabled: !user && isRegistrationReady }),
     trigger,
+    provider: chosenProvider,
   });
   const onProfileSuccess = async () => {
-    await refetchBoot();
+    const { data: boot } = await refetchBoot();
```

Comment:
Why do we need both this one as the onValidRegistration?
Seems like we could use one right?

Reply:
Will verify soon which ones we can let go and the ones we should keep.